Tools/autotest/quadplane.py

```diff
@@ -103,6 +103,40 @@ def get_disarm_delay(self):
     def set_autodisarm_delay(self, delay):
         self.set_parameter("LAND_DISARMDELAY", delay)
 
+    def test_motor_mask(self):
+        """Check operation of output_motor_mask"""
+        if not(int(self.get_parameter('Q_TILT_MASK')) & 1):
+#        or (int(self.get_parameter('Q_TAILSIT_MOTMX')) & 1)
+            self.progress("output_motor_mask not in use")
+            return
+        self.progress("Testing output_motor_mask")
+        self.reboot_sitl()
+        self.wait_ready_to_arm()
+
+        self.progress('Assert that SERVO5 is Motor1')
+        assert(33 == self.get_parameter('SERVO5_FUNCTION'))
+
+        self.progress("Testing MANUAL mode")
+        self.mavproxy.send('mode MANUAL\n')
+        self.wait_mode('MANUAL')
+        self.arm_vehicle()
+        self.progress("Raising throttle to 1800")
```

Comment:
Ah, a number to get out of date :-)

Reply:
Is there a "proper" way to make motors spin?

Reply:
Nah, just waiting for the numbers to diverge :-)